### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by bethiamchan**
This branch has:

## Implemented redux for user, movie, and filter states
image
image

## Added a case insensitive movie filter
image
image

### CODE REVIEW

File: src/components/main-view/main-view.jsx [hunk 1]
```diff
@@ -3,39 +3,32 @@ import axios from 'axios';
...
 import { ProfileView } from '../profile-view/profile-view';
-import { Nav } from 'react-bootstrap';
+
+import { setMovies, setUser } from '../../actions/actions';
+import MoviesList from '../movies-list/movies-list';
 
 export class MainView extends React.Component {
 	constructor() {
-		//Call superclass constructor so React can initialize it
 		super();
```
Comment (eadsimone) [inline src/components/main-view/main-view.jsx]: please add all props parameters for all super() function `Invalid number of arguments, expected 1` as `super(props);`

File: src/components/movies-list/movies-list.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	return (
+		<div className="movies-list">
+			<VisibilityFilterInput visibilityFilter={visibilityFilter} />
+			{filteredMovies.map((m) => (
+				<MovieCard key={m._id} movie={m} />
+			))}
+		</div>
+	);
+}
+
```
Comment (eadsimone) [inline src/components/movies-list/movies-list.jsx:32]: if going to be better add `mapStateToProps` and `mapdispatchtoprops` here. writing this code above the connect provide better Readability

**Review by eadsimone — CHANGES_REQUESTED** [review]
please in everywhere as _movie-view/movie-view.jsx_ that appears`<br></br>` replace with just `<br/>`
Everything looks great and working fine! did you submit in CF platform? I did not receive anything!

File: src/components/movies-list/movies-list.jsx (hunk 2 above)
Comment (bethiamchan) [reply src/components/movies-list/movies-list.jsx:32]: This has been updated.

File: src/components/main-view/main-view.jsx (hunk 1 above)
Comment (bethiamchan) [reply src/components/main-view/main-view.jsx]: This has been updated.

**Comment by bethiamchan** [comment]
The suggested changes have been made, and the logout flow is now working.

**Pull request merged**